sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -391,6 +391,29 @@ class HiveDDLSuite
     }
   }
 
+  test("create view with mismatched schema") {
```

Comment:
This is for `CREATE VIEW` Another is for `CREATE TEMP VIEW`. They are testing different code paths.